### ISSUE TITLE
terraform function gunzipbase64 as opposite of base64gzip is missing

### ISSUE THREAD

**Issue by l-with** [opentofu/opentofu#800]
### OpenTofu Version

```shell
v1.6.0-dev
```

### Use Cases

write base64gzip content to any storage, read it and decode it with gunzipbase64

### Attempted Solutions

none available

### Proposal

implement gunzipbase64

### References

https://github.com/hashicorp/terraform/issues/22568

**Comment by kislerdm** [comment]
Hey @l-with! Thank you for your proposal. Could you please share a few use cases to illustrate its impact and additional value?

We would like to share that it has decided to postpone execution beyond community discussions concerning this proposal until the stable version is released. The motivation: we would prefer to minimise deviation from terraform in terms of feature parity for the initial stable release. Thanks for understanding.

We noticed the effort which was already invested into development, it's very much appreciated! However, we would like to kindly remind you about the [contribution guidelines](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md). In particular, we would like to highlight the following rule of thumb.

----
### Please do NOT start working on the issue until it was accepted for development and labeled with the label `accepted`
----

Thank you!
